This file is part of MXE.
See index.html for further information.

Contains patch.

This patch has been taken from:
http://hg.orfeo-toolbox.org/OTB/rev/1524f80805f3

diff -r 878302156265 src/lib/openjp2/openjpeg.c
--- a/src/lib/openjp2/openjpeg.c	Wed Oct 01 14:18:25 2014 +0200
+++ b/src/lib/openjp2/openjpeg.c	Wed Oct 01 14:19:56 2014 +0200
@@ -130,7 +130,7 @@
 
 /* ---------------------------------------------------------------------- */
 #ifdef _WIN32
-#ifndef OPJ_STATIC
+#if !defined(OPJ_STATIC) && !defined(__MINGW32__)
 BOOL APIENTRY
 DllMain(HANDLE hModule, DWORD ul_reason_for_call, LPVOID lpReserved) {
 
